### ISSUE TITLE
Improve Tutorial page

### ISSUE THREAD

**Issue by rbonghi** [officinerobotiche/task-other#4]
I would like to improve the **Tutorial** page on master officinerobotiche website with an image and a little description for each pages in list in [tutorial](http://www.officinerobotiche.it/tutorial/) page.

I think to improve the user experience in the tutorial page, we can collect all similar pages in a topics.

cc @bobboteck

**Comment by bobboteck** [comment]
Yes is the final version I think for this page, actualy the plugin used non perform this request.
I'll try it one more suitable, otherwise I will have to try to develop a specific

**Comment by rbonghi** [comment]
:+1:

I would like to put my all italian pages in my blog in this page.
If you want I'm ready to help you
